lobsterpy/cohp/analyze.py

```diff
@@ -322,6 +333,158 @@ def get_information_all_bonds(self, summed_spins=True):
                     self.seq_labels_cohps.append(type_labels)
                     self.seq_cohps.append(type_cohps)
 
+    def get_site_bond_resolved_labels(self):
+        """
+
+        Returns:
+            dict with bond labels for each site, e.g.
+            {'Na1: Na-Cl': ['21', '23', '24', '27', '28', '30']}
+
+        """
+        bonds = [[] for _ in range(len(self.seq_infos_bonds))]  # type: ignore
+        labels = [[] for _ in range(len(self.seq_infos_bonds))]  # type: ignore
+        for inx, bond_info in enumerate(self.seq_infos_bonds):
+            for ixx, val in enumerate(bond_info.atoms):
+                label_srt = sorted(val.copy())
+                bonds[inx].append(
+                    self.structure.sites[bond_info.central_isites[0]].species_string
+                    + str(bond_info.central_isites[0] + 1)
+                    + ": "
+                    + label_srt[0].strip("0123456789")
+                    + "-"
+                    + label_srt[1].strip("0123456789")
+                )
+                labels[inx].append(bond_info.labels[ixx])
+
+        label_data = {}
+        for indx, atom_pairs in enumerate(bonds):
+            search_items = set(atom_pairs)
+            for item in search_items:
+                indices = [i for i, x in enumerate(atom_pairs) if x == item]
+                filtered_bond_label_list = [labels[indx][i] for i in indices]
+                label_data.update({item: filtered_bond_label_list})
+
+        return label_data
+
+    def _get_orbital_resolved_data(
+        self, nameion, iion, labels, bond_resolved_labels, type_pop

```

Comment:
Could we do snake-case where possible? It might be that I missed out on doing this for other methods before. However,  we could at least try to do it now ;).

Reply:
We can do this in another PR to not mix up things. The best would be to include this in documentation related PR I think😃